eng/versioning.targets

```diff
@@ -22,8 +22,8 @@
     </AssemblyMetadata>
   </ItemGroup>
 
-  <!-- Adds SupportedOSPlatform attribute for Windows Specific libraries and Windows targets -->
-  <ItemGroup Condition="('$(IsWindowsSpecific)' == 'true' or '$(TargetsWindows)' == 'true') and '$(IsTestProject)' != 'true'">
```

Comment:
As `SupportedOSPlatform("windows")` for windows targets will be added from MSBuild we don't need it anymore